models/opensea/ethereum/opensea_v3_ethereum_events.sql

```diff
@@ -34,12 +34,8 @@ WITH fee_wallets as (
 
 select *
 from trades
-where
-( zone_address in (0xf397619df7bfd4d1657ea9bdd9df7ff888731a11
-          ,0x9b814233894cd227f561b78cc65891aa55c62ad2
-          ,0x004c00500000ad104d7dbd00e3ae0a5c00560c00
-          ,0x110b2b128a9ed1be5ef3232d8e4e41640df5c2cd
-          ,0x000000e7ec00e7b300774b00001314b8610022b8 -- newly added on seaport v1.4
-          )
- or  fee_wallet_name = 'opensea'
+where (
+    fee_wallet_name = 'opensea'
+    or right_hash = 0x360c6ebe
```

Comment:
what does right hash mean and why this value?

Reply:
It's extra data that is added to the transaction to assist with onchain attribution.
It's `bytes4(keccak256("opensea.io"))`. More info here: https://docs.reservoir.tools/docs/calldata-attribution

Reply:
The other chain models for OS already use this filter, I'm not sure why ethereum was set up differently.